modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/service/impl/MotechSchedulerDatabaseServiceImpl.java

```diff
@@ -380,11 +389,18 @@ private String buildJobsBasicInfoSqlQuery(JobsSearchSettings jobsSearchSettings)
 
     private String buildJobsCountSqlQuery(JobsSearchSettings jobsSearchSettings) {
         StringBuilder sb = new StringBuilder("SELECT COUNT(*) FROM ");
-        sb = sb.append(getCorrectNameRepresentation(sqlProperties.get("org.quartz.jobStore.tablePrefix").toString() + TRIGGERS));
+        sb = sb.append(getCorrectNameRepresentation(sqlProperties.get(TABLE_PREFIX).toString() + TRIGGERS));
         sb = sb.append(buildWhereCondition(jobsSearchSettings));
         return sb.toString();
     }
 
+    private String buildJobsAllSqlQuery() {
+        StringBuilder sb = new StringBuilder("SELECT A.TRIGGER_NAME, A.TRIGGER_GROUP, B.JOB_DATA FROM ");
```

Comment:
bad using stringbuilder (I know that you did it similarly to others). Concat method from string creates new string instead of concating to existing one, however append from string builder is appending new string to the existing one.